dali/test/python/test_operator_erase.py

```diff
@@ -235,7 +241,14 @@ def test_operator_erase_with_out_of_bounds_roi_coords():
     axis_names = "HW"
     anchor_arg = (4, 10, 10, 4)
     shape_arg = (40, 50, 50, 40)
-    anchor_norm_arg = (4 / 60.0, 10 / 80.0, 2000, 2000, 10 / 60.0, 4 / 80.0)  # second region is completely out of bounds
+    anchor_norm_arg = (
+        4 / 60.0, 10 / 80.0, 2000, 2000, 10 / 60.0, 4 / 80.0
+    )  # second region is completely out of bounds
```

Comment:
Personal preference: I'd put the comment in the previous line and keep the tuple definition in one line

Reply:
done